src/core/render/index.js

```diff
@@ -128,18 +128,7 @@ export function renderMixin(proto) {
   };
 
   proto._renderSidebar = function(text) {
-    const { maxLevel, subMaxLevel, loadSidebar, hideSidebar } = this.config;
-
-    if (hideSidebar) {
-      // FIXME : better styling solution
-      document.querySelector('aside.sidebar').remove();
-      document.querySelector('button.sidebar-toggle').remove();
-      document.querySelector('section.content').style.right = 'unset';
-      document.querySelector('section.content').style.left = 'unset';
-      document.querySelector('section.content').style.position = 'relative';
-      document.querySelector('section.content').style.width = '100%';
```

Comment:
@anikethsaha Should we keep the style?

### develop branch

![image](https://user-images.githubusercontent.com/33931153/96199181-7bfc4a80-0f89-11eb-8f96-7386228f0a08.png)

### now 

![image](https://user-images.githubusercontent.com/33931153/96199129-5707d780-0f89-11eb-8b81-1dd2288ebb83.png)

Reply:
I think we can just keep this configurations without adjusting much styles right now （dont let render code including much styles）, and if there has much users proposal to make it that we can refine it again.
WDYT?

Reply:
I think we should keep them and not change them in the z versions.